galley/pkg/crd/validation/webhook.go

```diff
@@ -124,6 +124,9 @@ type WebhookParameters struct {
 
 	// Enable reconcile validatingwebhookconfiguration
 	EnableReconcileWebhookConfiguration bool
+
+	// Mux of an existing HTTP server, Galley will not manage its own
+	Mux *http.ServeMux
```

Comment:
It is no longer set - since Shriram asked to separate Galley-in-pilot to a different PR.

The change in Galley is still needed - to enable the webhook to be embedded in an existing server

Reply:
Please donot add currently unused stuff.